src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
Consider removing the `@Component` annotation from the interface and applying it to the implementation class instead. Annotations like `@Component` are generally used on concrete classes to indicate that they are candidates for dependency injection.